tests/system/providers/google/cloud/transfers/example_cloud_storage_transfer_service_gcp.py

```diff
@@ -142,13 +145,28 @@
         task_id="delete_transfer_from_gcp_job",
         job_name="{{task_instance.xcom_pull('create_transfer')['name']}}",
         project_id=GCP_PROJECT_ID,
+        trigger_rule=TriggerRule.ALL_DONE,
     )
 
-    chain(
```

Comment:
Just curious why not keep `chain()` here? You can still add the comments.

Reply:
Just to be consistent with the AIP-47 example